global.js

```diff
@@ -103,6 +103,8 @@ function onTextTyped (key) {
         }
       } else if (link.link.tagName === 'BUTTON') {
         link.link.click()
+      } else if (isCurrentlyInInput()) {
```

Comment:
why does this need to check `isCurrentlyInInput`?

Reply:
You're right this is an error - I've modified it in a more recent update.